src/game.cpp

```diff
@@ -3951,11 +3953,11 @@ void Game::handlePointingAtObject(const PointedThing &pointed, const ItemStack &
 			runData.object_hit_delay_timer = object_hit_delay;
 		}
 
-		if (getLeftClicked())
+		if (isKeyDown(KeyType::DIG))
 			do_punch = true;
 
 		if (do_punch) {
-			infostream << "Left-clicked object" << std::endl;
+			infostream << "Dug object" << std::endl;
```

Comment:
`Punched object`

Reply:
I've changed the string and renamed left_punch to just punch